.circleci/config.yml

```diff
@@ -9,7 +9,24 @@ jobs:
       - checkout
       - run:
           name: Acceptance Tests
-          command: "yarn install && yarn acceptance src/test/acceptance/*.js"
+          command: 'yarn install && yarn acceptance src/test/acceptance/*.js'
+  build:
+    docker:
+      - image: artsy/hokusai:0.4.5
+    steps:
+      - add_ssh_keys
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build
+          command: |
+            export commit_hash=`git rev-parse --short HEAD` && echo $commit_hash
+            hokusai build
+            docker save hokusai_force:latest -o hokusai_force.tar
```

Comment:
@joeyAghion wdty about enabling https://circleci.com/docs/2.0/docker-layer-caching/ - here we are working around always starting with a cold cache in the docker daemon itself by saving the built image and passing it along to subsequent build steps (which takes almost as long to restore as it takes to pull and rebuild the image) and doesn't allow us to simply rely on the built-in Docker caching to use a cached layer of `yarn install`, for instance, in subsequent builds...

Reply:
I think that enabling docker layer caching makes sense regardless but have you considered pushing/pulling to a registry instead of exporting/loading the image from disk? Utilizing a registry allows docker to pull layers selectively based on what it needs, might be a bit faster to start with, and would naturally benefit from access to a warm layer cache if we later enable the premium Circle CI feature.